### DISCUSSION TITLE
License of codesandbox-import-util-types and codesandbox-import-utils

### DISCUSSION THREAD

**Discussion by WangJiaoJo** [codesandbox/sandpack#984] [Q&A]
Hi, team,
We are using the **@codesandbox/sandpack-client** (1.19.0) and **@codesandbox/sandpack-react** (1.19.0) and there are two dependencies of them which are
**codesandbox-import-utils** 
**codesanbox-import-utils-types**,
and we find that the license of these two dependencies on **npmjs.com** is **none**.
But our Blackduck scan report the license risk about these two dependencies with **LGPL-3.0+**.
Is there any way that we can solve this license issue or do we need to buy the commercial license for code sandbox Sandpack? 
Thanks!

**Comment by WangJiaoJo** [comment]
Hi, team, @danilowoz 
Would you mind looking at this question? This would be much helpful if we can get your official reply.
Thanks in advance!

**Comment by Geminii** [comment]
Hi @WangJiaoJo 👋 
Do you have an update on this topic ? 🤔
